app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/filtersControl/filtersHeader/filtersHeader.scss

```diff
@@ -0,0 +1,20 @@
+.filters {
+  &-header {
```

Comment:
Do not use '&' to construct one class name. According to common style we are writing full class name.
But you still able to use construction like: &.additionalClassName, &:hover, &::after and ect.

Reply:
Please check and fix all other cases